dlc-manager/src/manager.rs

```diff
@@ -1231,6 +1231,117 @@ where
         Ok(())
     }
 
+    fn try_finalize_settled_closing_channel(
+        &self,
+        signed_channel: SignedChannel,
+    ) -> Result<(), Error> {
+        let (settle_tx, &is_offer, &is_initiator) = get_signed_channel_state!(
+            signed_channel,
+            SettledClosing,
+            settle_transaction,
+            is_offer,
+            is_initiator
+        )?;
+
+        if self
+            .blockchain
+            .get_transaction_confirmations(&settle_tx.txid())?
+            >= CET_NSEQUENCE
+        {
+            log::info!(
+                "Settle transaction {} for channel {} has enough confirmations to spend from it",
+                settle_tx.txid(),
+                serialize_hex(&signed_channel.channel_id)
+            );
+
+            let fee_rate_per_vb: u64 = {
+                let fee_rate = self
+                    .fee_estimator
+                    .get_est_sat_per_1000_weight(
+                        ConfirmationTarget::HighPriority,
+                    );
+
+                let fee_rate = fee_rate / 250;
```

Comment:
❓ why do we divide the fee rate by 250 here? I guess this constant should be defined as such.

Reply:
This is just copied code from another spot where we do the same thing. It's just the way we get a fee rate in sats/vB.